.editorconfig

```diff
@@ -0,0 +1,11 @@
+# editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# indent_size = 4
```

Comment:
Could this comment be removed? Or is there a point to it?

Reply:
It defines the codestyle used throughout the project. It's needed to make local clones use the correct format (otherwise you could get PRs using tabs, and other such things).

Reply:
I'm vaguely familiar with Editorconfig, it's just that lines beginning with `#` [are comments](https://editorconfig-specification.readthedocs.io/en/latest/#file-format), and I didn't see the reason to leave a line commented when it could just as well be uncommented or removed.

Reply:
Oh, you were talking about that. Yeah, that can be removed.